sdcm/cluster.py

```diff
@@ -4704,6 +4709,24 @@ def install_scylla_bench(node):
             GO111MODULE=on go get -v github.com/scylladb/scylla-bench@v0.1.2
         """))
 
+    @staticmethod
+    def install_cassandra_harry(node):
+        if node.distro.is_rhel_like:
+            node.remoter.sudo("yum install -y git make maven")
+        else:
+            node.remoter.sudo(shell_script_cmd("""\
+                apt-get update
+                apt-get install -y git make maven
+            """))
+        node.remoter.run(shell_script_cmd("""\
+            rm -rf cassandra-harry
+            git clone -b scylla-branch https://github.com/amoskong/cassandra-harry
```

Comment:
Please also ask Avi to move it into scylladb repos and change it here from "amosking" to scylladb.
Another option, is trying to contribute our changes to original apache repo.
Last option is use the original repo and apply the patches here (can break easily with changes but will be most up to date).

Reply:
But this can be in a following PR